Makefile

```diff
@@ -1627,6 +1627,12 @@ terraform-resources-up-to-date: must-start-clean/host
 		exit 1; \
 	fi
 
+# update-aws-regions-go-definition updates the AWS regions definition used by
+# golang based on the AWS SDK.
+update-aws-regions-go-definition:
+	@go generate ./lib/utils/aws/generate.go 
+	@echo "AWS regions definition updated!"
+
```

Comment:
can we make two targets, one for go:generate and one for CI?

example:
https://github.com/gravitational/teleport/blob/b587910e09a4683359deb64481ce2b524963b20e/Makefile#L1548-L1562

https://github.com/gravitational/teleport/blob/b587910e09a4683359deb64481ce2b524963b20e/.github/workflows/lint.yaml#L155-L157